seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -4349,11 +4353,11 @@
       "id": "56533eb9ac21ba0edf2244c9",
       "title": "Accessing Object Properties with Variables",
       "description": [
-        "Another use of bracket notation on objects is to use a variable to access a property. This can be very useful for iterating through lists of the object properties or for doing the lookup.",
+        "Another use of bracket notation on objects is to access a property which is stored as the value of a variable. This can be very useful for iterating through lists of the object properties or for doing the lookup.",
```

Comment:
I would change `"...iterating through lists of the object properties..."` 

to:

`"...iterating through an object's properties..."`

Reply:
What does "doing the lookup" refer to? Would it be better as `"when accessing a lookup table"`?